File: CS/Form1.cs
using System;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace dxSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }
        private void simpleButton1_Click(object sender, EventArgs e) {
            XtraReport report = new XtraReport1();
            new ReportHelper().CreateReportWithAutoCellWidth(report);
            report.ShowPreview();
        }
    }
}

File: VB/Form1.vb
Imports System
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace dxSample

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Dim report As XtraReport = New XtraReport1()
            Call New ReportHelper().CreateReportWithAutoCellWidth(report)
            report.ShowPreview()
        End Sub
    End Class
End Namespace
